### PR TITLE
Change Automatic Live View setting to allow for continuous playback

### PR DESCRIPTION
**Pull request by kensand**
Modify the Automatic Live View setting to allow users to enable continuous live streams in their dashboard and camera groups.

I'm happy to further refactor the `LivePlayer` to simplify its properties if desired; I just wanted to keep the changes small to start though.

Furthering the discussion in #12682. 

Tested by running `npm run dev` against my remote frigate instance with 8x720p cameras, and it runs great over a ~300 Mb connection. Also verified that Auto and Static modes still work.

### CODE REVIEW

**Comment by kensand** [comment]
Whoops, sorry; I think my IDE formatter got a little overzealous. I'll try to revert the whitespace changes (they can just be hidden in the meantime within Github's UI).

File: web/src/types/live.ts [hunk 1]
```diff
@@ -32,3 +32,6 @@ export type LiveStreamMetadata = {
 };
 
 export type LivePlayerError = "stalled" | "startup" | "mse-decode";
+
+export const LiveViewModes = ["Auto", "Static", "Continuous"] as const
```
Comment (NickM-27) [inline web/src/types/live.ts:36]: These should be lowercase to remain consistent with the other consts

File: web/src/views/settings/GeneralSettingsView.tsx [hunk 2]
```diff
@@ -52,52 +53,65 @@ export default function GeneralSettingsView() {
...
-                  detected. Disabling this option causes static camera images on
-                  the Live dashboard to only update once per minute.
-                </p>
-              </div>
+            <div className="text-md">Live View Modes</div>
+            <div className="my-2 text-sm text-muted-foreground">
+              <p>The mode for live streams. <br/> Auto mode (default) will begin streaming when motion is
+                detected.<br/> Static mode will update images on live streams once per minute.<br/> Continuous mode
+                will stream cameras regardless of motion. Caution: Continuous mode will increase bandwidth
+                usage and may affect performance.</p>
```
Comment (NickM-27) [inline web/src/views/settings/GeneralSettingsView.tsx:86]:
```suggestion
              <p>Live stream behavior:<br/>Auto mode (default) will begin streaming when activity is
                detected.<br/>Static mode will update camera images on the live dashboard once per minute.<br/>Continuous mode
                will stream cameras regardless of activity. Caution: Continuous mode will increase bandwidth usage and may affect performance.</p>
```

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Comment by hawkeye217** [comment]
Thanks for the PR. The code looks clean, good work.

Personally I am not sure this is the right approach to address the points you raised in the linked feature request (watching for motion occurring in a masked area, motion below threshold, etc). I won't repeat the points that @NickM-27 raised in the feature request, but as someone who works with Frigate users every day, I share his concerns. In my opinion, the potential for misconfiguration and excessive/unnecessary resource usage far outweighs the benefit for many users. 

Perhaps a more granular approach might be a better solution that could be of benefit to more users? I'd suggest creating a discussion to see if others have similar needs, and then looking at crafting something that keeps some of the main goals of Frigate - efficiency and portability -  in mind. We'd be happy to re-evaluate again at that point.

**Comment by kensand** [comment]
@hawkeye217 Thanks for having a look. I understand your and @NickM-27's  hesitancy to add options that could add to misconfiguration for users and increase your support workload. I will open a discussion as you suggest, and point to my fork for users in the meantime as a stopgap.

**Comment by kensand** [comment]
Created discussion #12697 to follow up.

**Comment by kensand** [comment]
Looks like github-actions closed this and I cannot re-open it. I didn't get a notification about the stale tag, so that's kinda annoying, but oh well :/

I won't bother creating a new PR until maintainers express a desire for this feature to be added, but I will keep my [dev branch](https://github.com/kensand/frigate/tree/dev) up-to date with this change so it can easily be PR'd again in the future anyways. The [main branch](https://github.com/kensand/frigate/tree/main) of my fork also includes this change, along with things necessary for packaging my own [images](https://github.com/kensand/frigate/pkgs/container/frigate) and any other improvements I make.

**Pull request closed without merging**